packages/contracts/tasks/deploy/maci/02-gatekeepers.ts

```diff
@@ -1,8 +1,9 @@
+import { hexToBigInt, uuidToBigInt } from "@pcd/util";
+
 import { HatsGatekeeperBase } from "../../../typechain-types";
 import { EDeploySteps, ESupportedChains } from "../../helpers/constants";
 import { ContractStorage } from "../../helpers/ContractStorage";
 import { Deployment } from "../../helpers/Deployment";
-import { uuidToBigInt } from "../../helpers/numericParser";
```

Comment:
If we don't use it anymore, let's remove the functions inside the file

Reply:
Thanks! fixed